### PR TITLE
Add Documentation for Expert Groups

### PR DESCRIPTION
**Pull request by klees**
Dear community,

in the past year, the [SIG Client](https://docu.ilias.de/goto_docu_grp_4941.html) had a series of workshops, using the technique of a "Zukunftswerkstatt". General goal of the workshops was to generate ideas how to move the client side of ILIAS into the future. One output of the session in [November 2019](https://docu.ilias.de/goto_docu_sess_8042.html) was the idea to create a group of UI/UX experts that were supposed to help on all stages of the development process with their knowledge.

This idea has been adopted by the [Technical Board](https://docu.ilias.de/goto_docu_grp_5089.html). We elaborated it into a general framework for experts in the various fields that are identified as the cross cutting concerns the TB is responsible of. We hereby want to present that general framework.

We expect, that a combined UI/UX/Accessibility expert group will be the first concrete group appointed in this framework and will get back to you soon with new results.

Until then we ask for your feedback regarding this framework and will bring it to the attention of the JF.

Best regards!

### CODE REVIEW

File: docs/development/expert-groups.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+**Expert Groups support the ILIAS community in improving quality aspects in the
+fields of usability, performance, security, software architecture and the
+development process.** These cross cutting concerns are the responsibility of
+the Technical Board, as assigned by the by-laws of the ILIAS society. Still,
+these fields cannot be treated effectively by the Technical Board alone but
+need broad consideration in all stages of the development process. The expert
+groups are meant to help the community and the Technical Board by providing
+knowledge and insights in various forms.
+
+To do that, **the groups will work on any means that serve the goal of improved
```
Comment (kergomard) [inline docs/development/expert-groups.md]:
```suggestion
To do that, **the groups will work on any means that serve the goal to improve
```

File: docs/development/expert-groups.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+quality e.g. consultancy, tools, code, concepts, documentation, guideline, process
+definitions.** Depending on the field, the state of the art in that field and the
+problem at hand, different approaches for improvements may be conceivable. The
+experts group are expected to identify and propose means that are effective,
```
Comment (kergomard) [inline docs/development/expert-groups.md]:
```suggestion
expert group are expected to identify and propose means that are effective,
```

File: docs/development/expert-groups.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+efficient and appropriate.
+
+To implement them, **the experts use existing processes for decision making, e.g.
+the Jour Fixe**, if required. They are also expected to use other processes and
+tools provided by the ILIAS community, such as the Mantis Issue Tracker, GitHub,
+the ILIAS conference and development conference. The experts should convince
```
Comment (kergomard) [inline docs/development/expert-groups.md]:
```suggestion
the ILIAS conference and the development conference. The experts should convince
```

File: docs/development/expert-groups.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+other community members by tangible reasoning, just as other community members,
+and have no special decision making power.
+
+To indicate a special trust of the TB, the **expert groups are appointed by the
+Technical Board**. This also gives the experts a special visibility in the
+community. **The members of the experts groups are appointed for their well-founded
+knowledge in the respective quality field**. They all are willing and able to
```
Comment (kergomard) [inline docs/development/expert-groups.md]:
```suggestion
knowledge in the respective field**. They all are willing and able to
```
Sounds bad and I don't think it is necessary.

**Review by kergomard — COMMENTED** [review]
Thank you very much @klees for this!
A few grammatical adaptations in the text.

Thanks again and best,
Stephan

**Comment by matthiaskunkel** [comment]
Jour Fixe, 11 JAN 2021 : We highly appreciate the suggested document and procedure. Thank you very much to the Technical Board. Please add document to the ILIAS documentation on GitHub.

**Pull request merged**